.gitignore

```diff
@@ -39,6 +42,7 @@ yarn-error.log
 testem.log
 /typings
 .reports/
+{}
```

Comment:
What does this ignore?

Reply:
Auto generated file created by the json-server.